src/features/intl-displaynames.md

```diff
@@ -0,0 +1,112 @@
+
+```js
+const scriptNames = new Intl.DisplayNames(['en'], { type: 'script' });
+scriptNames.of('Latn');
+// → 'Latin'
+scriptNames.of('Arab');
+// → 'Arabic'
+scriptNames.of('Kana');
+// → 'Katakana'
+```
+
+For more advanced usage, the second `options` parameter also supports the `style` property. The `style` property corresponds to the width of the display name and may be either `"long"`, `"short"`, or `"narrow"`. The values for different styles do not always differ. The default is `"long"`.
+
+```js
+const longLanguageNames = new Intl.DisplayNames(['en'], { type: 'language' });
+longLanguageNames.of('en-US');
+// → 'American English'
+const shortLanguageNames = new Intl.DisplayNames(['en'], { type: 'language', style: 'short' });
+shortLanguageNames.of('en-US');
+// → 'US English'
+const narrowLanguageNames = new Intl.DisplayNames(['en'], { type: 'language', style: 'narrow' });
+narrowLanguageNames.of('en-US');
+// → 'US English'
+```
+
+## Full API
+
+The full API for `Intl.DisplayNames` is as follows.
+
+```js
+Intl.DisplayNames([ locales [ , options ]])
+Intl.DisplayNames.prototype.of( code )
+```
+
+The constructor is consistent with other `Intl` APIs. Its first argument is a [list of locales](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation), and its second parameter is an `options` parameter that takes `localeMatcher`, `type`, and `style` properties.
+
+The `"localeMatcher"` property is treated the same as in [other `Intl` APIs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation). The `type` property may be `"region"`, `"language"`, `"currency"`, or `"script"`. The `style` property may be `"long"`, `"short"`, or `"narrow"`, with `"long"` being the default.
+
+`Intl.DisplayNames.prototype.of( code )` expects the following formats depending on the `type` of how the instance is constructed.
+
+- When `type` is `"region"`, `code` must be either an [ISO-3166 2-letter country code](https://www.iso.org/iso-3166-country-codes.html) or a [UN M49 3-digit region code](https://unstats.un.org/unsd/methodology/m49/).
+- When `type` is `"language"`, `code` must be conform to [Unicode's language identifier grammar](https://unicode.org/reports/tr35/#Unicode_language_identifier).
+- When `type` is `"currency"`, `code` must be a [ISO-4217 3-letter currency code](https://www.iso.org/iso-4217-currency-codes.html).
+- When `type` is `"script"`, `code` must be a [ISO-15924 4-letter script code](https://unicode.org/iso15924/iso15924-codes.html).
+
+## Conclusion
+
+Like other `Intl` APIs, as `Intl.DisplayNames` become more widely available, libraries and applications will opt to drop packaging and shipping their own translation data in favor of using the native functionality.
+
+## `Intl.DisplayNames` support { #support }
+
+<feature-support chrome="81 /blog/v8-release-81#intl.displaynames"
```

Comment:
Note that the V8 v8.1 release blog post hasn't happened yet. @natorion What's the status on that?

Reply:
Missed that this one produces a broken table; on it.